src/lib/intlFormat.test.ts

```diff
@@ -202,55 +202,49 @@ describe('intlFormat', () => {
     });
   });
 
+  const date = DateTime.local(2020, 1, 5);
+
   describe('dateFormat', () => {
     it('formats day and month as date', () => {
-      expect(dateFormat(DateTime.local(2020, 1, 5), 'en-US')).toEqual(
-        'Jan 5, 2020',
+      expect(dateFormat(date, 'en-US')).toEqual('Jan 5, 2020');
+    });
+
+    it('formats full date', () => {
+      expect(dateFormat(date, 'en-US', { fullMonth: true })).toEqual(
```

Comment:
[Nice-to-have] Good test coverage for new fullMonth option. Evidence: Test verifies `dateFormat(date, 'en-US', { fullMonth: true })` returns 'January 5, 2020'. Impact: Ensures functionality works correctly. [Category: File]